src/NLog.Targets.Syslog/Settings/SdElementConfig.cs

```diff
@@ -13,13 +13,13 @@ namespace NLog.Targets.Syslog.Settings
     /// <summary>Syslog SD-ELEMENT field configuration</summary>
     public class SdElementConfig : NotifyPropertyChanged, IDisposable
     {
-        private SdIdConfig sdId;
+        private Layouts.SimpleLayout sdId;
```

Comment:
The implicit operator was present in my SdIdConfig, but I think that using SimpleLayout is better.
I simply would not prefix it with the `Layouts.` qualifier